manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司
+PackageName: KRInstall PGR Overseas

```

```suggestion
PackageName: Punishing Gray Raven
```
This should match control panel